spec/abstractops.html

```diff
@@ -1709,16 +1710,22 @@ <h1>
     </emu-alg>
   </emu-clause>
 
-  <emu-clause id="sec-temporal-preparetemporalfields" aoid="PrepareTemporalFields">
-    <h1>PrepareTemporalFields ( _fields_, _fieldNames_, _requiredFields_ )</h1>
-    <p>
-      The abstract operation PrepareTemporalFields reads the relevant properties of an Object _fields_; ensures all the required properties are present; and creates a new Object with all the properties converted to the correct type.
-      The relevant property names are given by the List of Strings _fieldNames_, and the required property names are given by the List of Strings _requiredFields_.
-      It performs the following steps:
-    </p>
-    <emu-note>The values of ? ToIntegerThrowOnInfinity(*undefined*) and ? ToPositiveInteger(*undefined*) are 0.</emu-note>
+  <emu-clause id="sec-temporal-preparetemporalfields" type="abstract operation">
+    <h1>
+      PrepareTemporalFields (
+        _fields_: an Object,
+        _fieldNames_: a List of property names,
+        _requiredFields_: a List of property names,
```

Comment:
does "property names" constrain it to being strings? or should this be Property Keys?

Reply:
"Property name" is defined here: https://tc39.es/ecma262/#property-name
It's constrained to strings, which seems correct to me. We don't currently allow calendar field names that are Symbols. (See the IterableToListOfType validation step in [CalendarFields](https://tc39.es/proposal-temporal/#sec-temporal-calendarfields).) In the unlikely event that Symbols somehow became necessary in the future it would be possible to introduce it later.

Reply:
ah, didn't realize it was defined, sounds good then